tests/helpers/test_config_validation.py

```diff
@@ -299,16 +299,16 @@ def test_template():
     """Test template validator."""
     schema = vol.Schema(cv.template)
 
-    for value in (
-        None, '{{ partial_print }', '{% if True %}Hello', {'dict': 'isbad'}
-    ):
+    for value in (None, '{{ partial_print }', '{% if True %}Hello'):
         with pytest.raises(vol.MultipleInvalid):
             schema(value)
 
     for value in (
         1, 'Hello',
         '{{ beer }}',
         '{% if 1 == 1 %}Hello{% else %}World{% endif %}',
+        {'test': 1, 'test': '{{ beer }}'},
+        ['{{ beer }}', 1]
```

Comment:
A question on the coverage. Why would this test not be covering [this](https://coveralls.io/builds/7787312/source?filename=homeassistant%2Fhelpers%2Fservice.py#L69) line?